### PR TITLE
Update DG use cases and UG 'add' feature

### LINKED ISSUE #5: Add "Add/Create" feature to User Guide

### LINKED ISSUE #9: Add Use Cases to Developer's guide
Give use cases (textual form) for a few representative user stories that need multiple steps to complete. e.g. Adding a tag to a person (assume the user needs to find the person first)

### PR DESCRIPTION
**Pull request by tengjianling**
Update use cases in Developer Guide and 'add' feature in User Guide, as discussed for week 7 tP tasks

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -262,30 +262,141 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    Use case resumes at step 2.
+
+- 3b. The edit request is invalid
+
+  - 3b1. Medmoriser shows an error message
+
+    Use case resumes at step 2.
+
+Use case: UC06 - Find a Q&A
```
Comment (jonfoocy) [inline docs/DeveloperGuide.md]: Might have missed out on bolding lines 383, 385, 393

**Review by jonfoocy — DISMISSED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tengjianling) [reply docs/DeveloperGuide.md]: Ok I bolded the 3 lines

**Review by jonfoocy — APPROVED** [review]
LGTM 💯

**Review by joshruien — APPROVED** [review]
Looks good!

**Pull request merged**